config/src/main/java/com/alibaba/nacos/config/server/controller/CapacityController.java

```diff
@@ -101,13 +100,13 @@ public CapacityController(CapacityService capacityService) {
             capacityService.initAllCapacity();
             RestResult<Boolean> restResult = new RestResult<Boolean>();
             setFailResult(response, restResult, 400);
-            restResult.setMessage("参数group和tenant不能同时为空");
+            restResult.setMessage("The parameters Group and Tenant cannot both be empty");
             return restResult;
         }
         if (quota == null && maxSize == null && maxAggrCount == null && maxAggrSize == null) {
             RestResult<Boolean> restResult = new RestResult<Boolean>();
             setFailResult(response, restResult, 400);
-            restResult.setMessage("参数quota、maxSize、maxAggrCount、maxAggrSize不能同时为空");
+            restResult.setMessage("quota max size max aggr count max aggr size");
```

Comment:
The first word of the sentence should be capitalized

Reply:
0xa36fD8fF0524f287422337d99bABdf05923f581B

Reply:
```suggestion
            restResult.setMessage("quota max size max aggr count max aggr size");0xa36fD8fF0524f287422337d99bABdf05923f581B
```